torch/_jit_internal.py

```diff
@@ -983,9 +985,12 @@ def is_tuple(ann) -> bool:
     # For some reason Python 3.7 violates the Type[A, B].__origin__ == Type rule
     if not hasattr(ann, "__module__"):
         return False
+
+    ann_origin = getattr(ann, "__origin__", None)
+    if IS_PY39_PLUS and ann.__module__ == "builtins" and ann_origin is tuple:
+        return True
     return ann.__module__ == "typing" and (
```

Comment:
Remove the parenthesis?

Reply:
I need parenthesis for `or` to take precedence, otherwise linter suggests the same ;)